cli/sql_util_test.go

```diff
@@ -74,15 +74,15 @@ OK
 		t.Fatal(err)
 	}
 
-	expectedCols := []string{"Field", "Type", "Null"}
+	expectedCols := []string{"Field", "Type", "Null", "Default"}
 	if !reflect.DeepEqual(expectedCols, cols) {
 		t.Fatalf("expected:\n%v\ngot:\n%v", expectedCols, cols)
 	}
 
 	expectedRows := [][]string{
-		{`"parentID"`, `"INT"`, `true`},
-		{`"name"`, `"STRING"`, `true`},
-		{`"id"`, `"INT"`, `true`},
+		{`"parentID"`, `"INT"`, `true`, `<nil>`},
```

Comment:
looking at this again, we should print `NULL` as `NULL`. Seems like that would go in `sqlRowsToStrings`